Sprint-2/debug/2.js

```diff
@@ -12,3 +12,23 @@ console.log(`The last digit of 806 is ${getLastDigit(806)}`);
 
 // This program should tell the user the last digit of each number.
 // Explain why getLastDigit is not working properly - correct the problem
+
+It is not working properly because only one varibale was declared, wich is num, 
```

Comment:
**What do we call this type of variable, declared outside the function in this way?**
It's called a global variable.

**What is the name of the type of word inside the parentheses in the function?**
an expression


Conclusion - So my answer should have been, the reason it is not working is you can't pass a global variable as an expression because the function creates its own space, away from the global variable which will allow us to declare local variables.  

The expressions we pass on to the parenthesis can then become our local variables, if we apply the logic of the above function we will get to a point where our local and global variables have the same name.